models/public/GPT-2/README.md

```diff
@@ -0,0 +1,85 @@
+# GPT-2
+
+## Use Case and High-Level Description
+
+The `GPT-2` model is a transformers model pre-trained on a very large corpus of English data in a self-supervised fashion. GPT-2 is trained with a simple objective: predict the next word, given all of the previous words within some text.
+
+More details provided in the [paper](https://cdn.openai.com/better-language-models/language_models_are_unsupervised_multitask_learners.pdf), [repository](https://github.com/huggingface/transformers) and [model card](https://huggingface.co/gpt2).
+
+## Specification
+
+| Metric            | Value            |
+|-------------------|------------------|
+| Type              | Text Prediction  |
+| GFlops            | 293.0489         |
+| MParams           | 175.6203         |
+| Source framework  | PyTorch\*        |
+
+## Accuracy
+
+Accuracy metrics obtained on [WikiText-2 raw character level data](https://blog.einstein.ai/the-wikitext-long-term-dependency-language-modeling-dataset/) dataset for converted model.
+
+| Metric     | Value  |
+| ---------- | ------ |
+| Perplexity | 29.00% |
+
+## Input
+
+### Original model
+
+Image, name: `input`, shape: `1, 1024` in the format `B, L`, where:
+
+- `B` - batch size
+- `L` - sequence length
+
+### Converted model
+
+Image, name: `input`, shape: `1, 1024` in the format `B, L`, where:
+
+- `B` - batch size
+- `L` - sequence length
+
+## Output
+
+### Original model
+
+1. Prediction scores of language modeling head, name: `output`, shape: `1, 1024, 50257` in the format `B, L, S`, where:
+
+    - `B` - batch size
+    - `L` - sequence length
+    - `S` - vocab size
+
+2. Pre-computed hidden states, names: `274, 289, 503, 518, 732, 747, 961, 976, 1190, 1205, 1419, 1434, 1648, 1663, 1877, 1892, 2106, 2121, 2335, 2350, 2564, 2579, 2793, 2808`, shape: `1, 12, 1024, 64` in format `B, N, L, S`, where:
```

Comment:
do we need them or you can just remove them during  postprocessing?

Reply:
These outputs are removed during conversion to IR.

Reply:
yes, but you can align behaviour for onnx too, just remove from model code return of these layers

Reply:
it allow your demo working with both model representation

Reply:
Done